subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -23,9 +23,11 @@ static int init_reset(void);
  */
 int lll_df_init(void)
 {
+#if defined(CONFIG_BT_CTLR_DF_ANT_SWITCH_TX) || defined(CONFIG_BT_CTLR_DF_ANT_SWITCH_RX)
```

Comment:
This should be not required since: 
https://github.com/zephyrproject-rtos/zephyr/blob/4aed856d7fb17ced2554aaebe5a3c3069d127a74/subsys/bluetooth/controller/Kconfig.df#L122-L124